labs/lab17-persistent-storage/DBClientApp/manifests/db-consumer.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: repl-db-consumer
+  labels:
+    app: repl-db-consumer
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: repl-db-consumer
+  template:
+    metadata:
+      labels:
+        app: repl-db-consumer
+    spec:
+      containers:
+        - name: repl-db-consumer
+          image: cremerfc/repl-db-consumer:latest
```

Comment:
Can we use ghcr?
https://github.com/replicatedhq/kots-field-labs/blob/main/.github/workflows/lab11-monitoring.yml#L54